system-addon/jar.mn

```diff
@@ -17,13 +17,28 @@
   content/vendor/react-intl.js (./vendor/react-intl.js)
   content/vendor/redux.js (./vendor/redux.js)
   content/vendor/react-redux.js (./vendor/react-redux.js)
-  content/data/ (./data/*)
+  content/data/content/locales.json (./data/locales.json)
```

Comment:
`locales.json` doesn't actually make it to mozilla-central

Reply:
whoops, wrong path!